devops/dev/nginx.conf

```diff
@@ -2,7 +2,7 @@ server {
     listen 80;
 
     location /api/ {
-        proxy_pass         http://backend:4000/;
+        proxy_pass         http://backend:8000/;
```

Comment:
This is not a good idea for the production environment ... We should stick with 4000